src/main/java/net/unicon/lti/service/lti/impl/RegistrationServiceImpl.java

```diff
@@ -90,5 +98,25 @@ public String callDynamicRegistration(String token, ToolRegistrationDTO toolRegi
         return answer;
     }
 
+    @Override
+    public void saveNewPlatformDeployment(ToolConfigurationACKDTO answer, PlatformRegistrationDTO platformRegistrationDTO) throws RegistrationException {
+        try {
+            PlatformDeployment platformDeployment = new PlatformDeployment();
+            platformDeployment.setDeploymentId(answer.getToolConfiguration().getDeployment_id());
+            platformDeployment.setClientId(answer.getClient_id());
+            platformDeployment.setIss(platformRegistrationDTO.getIssuer());
+            platformDeployment.setJwksEndpoint(platformRegistrationDTO.getJwks_uri());
+            platformDeployment.setoAuth2TokenAud(platformRegistrationDTO.getToken_endpoint());
```

Comment:
I don't think this will work for D2L. I think D2L requires the oAuth2TokenAud to be equal to the authorization_server value.